packages/accounts-app/src/Accounts/BalanceOverview.tsx

```diff
@@ -75,29 +71,34 @@ export function BalanceOverview (props: Pick<Props, Exclude<keyof Props, keyof '
       amountAsString: unbondAmount.toString(),
       accountNonce: controllerNonce,
       currentBalance: controllerBalance,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
       // @ts-ignore
```

Comment:
dumb question but does `ts-ignore` make sense anymore with eslint?

Reply:
We still use `tsc`, so we still need ts-ignore to ignore those